core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -1328,6 +1328,8 @@ private void processResponse(org.jgroups.Address src, byte[] buffer, int offset,
          requests.addResponse(requestId, address, response);
       } catch (Throwable t) {
          log.errorProcessingResponse(requestId, src);
+         Exception e = t instanceof Exception ? ((Exception) t) : new CacheException(t);
```

Comment:
This change seems unrelated? What is this a fix for?

Reply:
I was seeing a testsuite stall due to a bug (during development) causing NPE. The bug has been fixed, so this is not technically needed...